tests/test_full_dataset.py

```diff
@@ -92,7 +92,7 @@ def test_load_mtracks(skip_remote, dataset):
 
     assert isinstance(all_data, dict)
 
-    mtrack_ids = dataset.mm
+    mtrack_ids = dataset.mtrack_ids
```

Comment:
Good catch, and sorry about that. We are going to be adding a fix for that in a separate PR here: #657.